packages/aws-lambda/layer/bin/publish-layer.sh

```diff
@@ -315,54 +283,53 @@ popd > /dev/null
 export AWS_PAGER=""
 export AWS_MAX_ATTEMPTS=$AWS_CLI_RETRY_MAX_ATTEMPTS
 
-if [[ -z "$BUILD_X86_64_FAIL" ]]; then
-    export BUILD_X86_64_FAIL=0
-fi
-
-if [[ -z "$BUILD_arm64_FAIL" ]]; then
-    export BUILD_arm64_FAIL=0
-fi
-
 if [[ -z $SKIP_AWS_PUBLISH_LAYER ]]; then
   echo "step 6/9: publishing $ZIP_NAME as AWS Lambda layer $LAYER_NAME to specifed regions"
+  
+  if [[ "$PUBLISH_TO_CHINA_REGIONS" == "true" ]]; then
```

Comment:
suggestion: You could also remove the list of CHINESE_REGIONS on top. And make it dynamic. This will work out of the box if new cn regions are added.

```bash
REGIONS=$(aws ssm get-parameters-by-path --path /aws/service/global-infrastructure/services/lambda/regions --output text --query "Parameters[].Value" | tr '\t' '\n' | sort)

REGIONS_CN=()
REGIONS_OTHERS=()

while IFS= read -r region; do
    if [[ $region == cn-* ]]; then
        REGIONS_CN+=("$region")
    else
        REGIONS_OTHERS+=("$region")
    fi
done <<< "$REGIONS"
```

Reply:
Thank you for the suggestion ❤️, I’ve implemented it!